kvui.py

```diff
@@ -836,6 +836,9 @@ def _handle_color(self, node: JSONMessagePart):
         return self._handle_text(node)
 
     def _handle_text(self, node: JSONMessagePart):
+        node_type = node.get("type", None)
+        if node_type == "text" or node_type == None: # All other text goes through _handle_color, and we don't want to escape markup twice
+            node["text"] = escape_markup(node["text"])
```

Comment:
```suggestion
        if node_type == "text" or node_type == None:
            # All other text goes through _handle_color, and we don't want to escape markup twice
            node["text"] = escape_markup(node["text"])
```
Not sure if you like the comment there, or if you'd prefer it on the line before, or shorten it somehow, but your version violates the 120 line length limit.

Would recommend using pycharm-community (an an IDE with linter) if you plan on committing more/bigger changes, since that would've shown 2 style warnings: 2 spaces before `#` and line length <= 120. See also our [style guide](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md#python-code).

Reply:
Ok, I think I made it so the comment follows the style guide? I'm going to also try to install PyCharm as well.

https://github.com/ArchipelagoMW/Archipelago/pull/3659/commits/99105a9a1a35c9727d71762c89a4b0007cdab22e